tools/slackbot/src/release-bot/release-bot.types.ts

```diff
@@ -4,7 +4,9 @@ export const Channels = {
   'leafygreen-ui-releases': 'C01CBLPFD35',
 } as const;
 
-export interface Opts {
+export type ChangeType = 'major' | 'minor' | 'patch' | 'dependency';
```

Comment:
Is the `dependency` type new?

Reply:
It's just a special designation we have we want to format them differently. `dependency` change type is just a patch with nothing other than dependency bumps